ext/php8/excluded_modules.c

```diff
@@ -18,11 +18,6 @@ bool ddtrace_is_excluded_module(zend_module_entry *module, char *error) {
         PHP 7.0 was only supported from Xdebug 2.4 through 2.7
         @see: https://xdebug.org/docs/compat
         */
```

Comment:
This comment can be removed since it refers to the removed code below.

Reply:
thanks for the catch https://github.com/DataDog/dd-trace-php/pull/1165/commits/b6fbc66acb27985a8316299102af59576b8516b2